src/logger.ts

```diff
@@ -21,6 +22,13 @@ interface LogEvent {
   vercel?: VercelData;
 }
 
+export enum LogLevel {
+  debug = 0,
+  info = 1,
+  warn = 2,
+  error = 3,
+}
```

Comment:
Could we use string enums here?

Reply:
We can, but what would be the way to check which level is higher/lower than the configured level?